test/unit-test/test-Model.js

```
@@ -0,0 +1,851 @@
+      return nn.createModel().then((model)=>{
+        assert.throws(() => {
+          model.addOperand();
+        });
+      });
+    });
+
+    it('raise error when passing parameter is not "OperandOptions" type', function() {
+      return nn.createModel().then((model)=>{
+        assert.throws(() => {
+          model.addOperand(123);
+        });
+      });
+    });
+
+    it('raise error when passing two parameters', function() {
+      return nn.createModel().then((model)=>{
+        assert.throws(() => {
+          model.addOperand({type: nn.INT32}, {type: nn.INT32});
+        });
+      });
+    });
+  });
+
+  describe('#setOperandValue API', function() {
+    it('check "setOperandValue" is a function', function() {
+      return nn.createModel().then((model)=>{
+        assert.equal(typeof model.setOperandValue, 'function');
+      });
+    });
+
+    it('check return value is of "void" type', function() {
+      return nn.createModel().then((model)=>{
+        model.addOperand({type: nn.INT32});
+        assert.equal(model.setOperandValue(0, new Int32Array([nn.FUSED_NONE])), undefined);
+      });
+    });
+
+    it('target operand type is FLOAT32 (0), setting its value as a Int8Array is ok', function() {
+      return nn.createModel().then((model)=>{
+        model.addOperand({type: nn.FLOAT32, dimensions: [1, 2, 2, 1], scale: 0.8, zeroPoint: 1});
+        let inputData = new Int8Array(4);
+        inputData.fill(0);
+        assert.doesNotThrow(() => {
+          model.setOperandValue(0, inputData);
+        });
+      });
+    });
+
+    it('target operand type is FLOAT32 (0), setting its value as a Uint8Array is ok', function() {
+      return nn.createModel().then((model)=>{
+        model.addOperand({type: nn.FLOAT32, dimensions: [1, 2, 2, 1], scale: 0.8, zeroPoint: 1});
+        let inputData = new Uint8Array(4);
+        inputData.fill(0);
+        assert.doesNotThrow(() => {
+          model.setOperandValue(0, inputData);
```

This should throw invalid typed array error.